script/remove_offline_runners.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Canonical
+# See LICENSE file for licensing details.
+
+import sys
+import logging
+
+import requests
+
+ORG = "<your org>"
+TOKEN = "<your token>"
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def get_runners():
+    try:
+        response = requests.get(
```

Comment:
requests does not have a default timeout, without timeout requests might wait indefinitely.